snarkos/store/mod.rs

```diff
@@ -85,4 +85,6 @@ pub enum DataID {
     KeyValueIDMap,
     KeyMap,
     ValueMap,
+    // A value for testing purposes
+    Test,
```

Comment:
Generally don't like sprinkling feature flags everywhere, but we should only expose the `Test` enum in the testing context. 

Adding `#[cfg(test)]` might be sufficient